vendor/raylib/raygui.odin

```diff
@@ -0,0 +1,560 @@
+package raylib
+
+import c "core:c/libc"
+
+when ODIN_OS == .Windows {
+	foreign import lib {
+		"windows/raygui.lib",
+	}
+} else when ODIN_OS == .Linux  {
+	foreign import lib { 
+		"linux/libraygui.a",
+		// "system:dl",
+		// "system:pthread",
+	}
+} else when ODIN_OS == .Darwin {
+	when ODIN_ARCH == .arm64 {
+		foreign import lib {
+			"macos-arm64/libraygui.a",
+			// "system:Cocoa.framework",
+			// "system:OpenGL.framework",
+			// "system:IOKit.framework",
+		}
+	} else {
+		foreign import lib {
+			"macos/libraygui.a",
+			// "system:Cocoa.framework",
+			// "system:OpenGL.framework",
+			// "system:IOKit.framework",
+		}
+	}
+} else {
+	foreign import lib "system:raygui"
+}
+
+RAYGUI_VERSION :: "4.0"
+
+// Style property
+GuiStyleProp :: struct {
+    controlId: u16,
+    propertyId: u16,
+    propertyValue: c.int,
```

Comment:
Please use tabs for indentation.

Reply:
Yeah, sorry for that, my editor was quite unhelpful getting this right. Should be right now, please check.